### DISCUSSION TITLE
What is scrollOffset for exactly?

### DISCUSSION THREAD

**Discussion by adiba** [faceless-ui/slider#19] [General]
Is the purpose similar to that of https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-padding ?

**Comment by jacobsfletch** [comment]
@adiba yea pretty much spot on by the looks of it. Imagine you have a full width slider that extends to each side of the viewport, but you don't want the slides themselves to be fully left-aligned with the viewport, instead you want them aligned with the content of your page, for example. This is the use-case for that, its very helpful in aligning the slides with your own grid.
